sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.cpp

```diff
@@ -1269,6 +1205,76 @@ ur_result_t _ur_device_handle_t::initialize(int SubSubDeviceOrdinal,
     ZeEventsScope = DeviceEventsSetting;
   }
 
+  uint32_t numQueueGroups = 0;

```

Comment:
why did you move this code? is it changed in any way?

Reply:
The two blocks are unchanged but rearranged with respect to each other. The old upper part leads to a query for isPVC(), to set limits for copy engines. At that time the device properties are not initialized. By rearranging the blocks the call to isPVC() works as expected. Otherwise there are references to uninitilaized values and a crash.